### DISCUSSION TITLE
Google Colab Notebook URL

### DISCUSSION THREAD

**Discussion by shanmugad** [vladmandic/sdnext#1486] [Q&A]
Hi

Can anyone share Google Colab Notebook URL from this Git repo?

**Comment by 1r00t** [comment]
You can find a link here:
https://github.com/camenduru/stable-diffusion-webui-colab#-special-colab

I tried it yesterday, it was not working as is, because it was using an older torch version or something.
You'd have to update the line where it installs torch. I used the one from here:
https://pytorch.org/get-started/pytorch-2.0/#faqs
Currently it lists this:
```bash
pip3 install numpy --pre torch torchvision torchaudio --force-reinstall --index-url https://download.pytorch.org/whl/nightly/cu118
```

**Comment by toyssamurai** [comment]
I think you could just make one up without much problem. I am not using Google Colab but Paperspace, all I need to do to get it running is add a cell to an empty notebook and do:

!cd /notebooks
!git clone https://github.com/vladmandic/automatic
!/notebooks/sdnext/webui.sh --auth=abc:123 --share --no-download

NOTE: I need to run !/notebooks/sdnext/webui.sh 2 times to workaround the "deprecated" import error.

**Comment by djwashout** [comment]
Are you needing a google colab notebook to run the repo?
